### PR TITLE
added aggregation example

### PR DESCRIPTION
**Pull request by albertlockett**
Added example of how to use aggregations. 

I moved the examples each into their own folder, so that there aren't multiple files in the package with `main` function

### CODE REVIEW

File: examples/aggregations/aggregations.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+		WeatherRecord{Day: "Tue", Snowfall: 15, City: minneapolis},
+		WeatherRecord{Day: "Wed", Snowfall: 32, City: minneapolis},
+		WeatherRecord{Day: "Thu", Snowfall: 10, City: minneapolis},
+		WeatherRecord{Day: "Fri", Snowfall: 12, City: minneapolis},
+	)
+
+	// Create a new query engine to retrieve data
+	engine := query.NewEngine(memory.DefaultAllocator, database.TableProvider())
+
+	// Weekly snowfall statistics by city:
```
Comment (teleivo) [inline examples/aggregations/aggregations.go:78]: When reading `Weekly` I thought you are aggregating per week. Now I see that the data is only for a week. Maybe remove the word to reflect the query 😄

**Review by teleivo — COMMENTED** [review]
Nice 😄 I am a frostdb beginner and created my own aggregation example. I think it's a helpful example to have. I love the frosty theme ☃️

File: examples/aggregations/aggregations.go [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+func main() {
+
+	// Create a new column store
+	columnstore, _ := frostdb.New()
+	defer columnstore.Close()
+
+	// Open up a database in the column store
+	database, _ := columnstore.DB(context.Background(), "weather_db")
+
+	// Define our simple schema of labels and values
```
[the remark this answers is not in the document]
Comment (albertlockett) [reply examples/aggregations/aggregations.go]: thanks @teleivo :)

File: examples/aggregations/aggregations.go (hunk 1 above)
Comment (albertlockett) [reply examples/aggregations/aggregations.go]: Ah thanks @teleivo -- I can see how it might be confusing. I have updated the wording :+1:

**Comment by thorfour** [comment]
Awesome! Thank you for the contribution

**Pull request merged**